site/app/libraries/database/DatabaseQueries.php

```diff
@@ -2561,7 +2561,7 @@ public function getUsersByRotatingSections($sections, $orderBy = "rotating_secti
      * @return array
      */
     public function getRegistrationSections() {
-        $this->course_db->query("SELECT * FROM sections_registration ORDER BY SUBSTRING(sections_registration_id, '^[^0-9]*'), COALESCE(SUBSTRING(sections_registration_id, '[0-9]+')::INT, -1), SUBSTRING(sections_registration_id, '[^0-9]*$') ");
+        $this->course_db->query("SELECT * FROM sections_registration ORDER BY SUBSTRING(sections_registration_id, '^[^0-9]*'), CAST(SUBSTRING(sections_registration_id, '[0-9]+') AS NUMERIC), SUBSTRING(sections_registration_id, '[^0-9]*$');");
```

Comment:
There are two semi colons, do we need the semi colon inside the double quotation?